### PR TITLE
Dat 3693 extract jsonformat tests to separate suite

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

File: extensions/wikia/JsonFormat/tests/DivContainingHeadersVisitorTest.php [hunk 1]
```diff
@@ -10,6 +10,7 @@ public function setUp() {
 	}
 
 	/**
+	 * @group ExternalDependencyIntegration
```
Comment (sebastianmarzjan) [inline extensions/wikia/JsonFormat/tests/DivContainingHeadersVisitorTest.php]: ContractualResponsibilitiesValidation ?

File: tests/Makefile [hunk 2]
```diff
@@ -41,6 +41,11 @@ phpunit-externals: prepare
 	${CMD_PHPUNIT} \
 		--group ExternalIntegration
 
+# runs tests for externally used parts
+phpunit-dependents: prepare
```
Comment (sebastianmarzjan) [inline tests/Makefile]: phpunit-contractual-responsibilities ?

**Comment by sebastianmarzjan** [comment]
Is it possible to add to the asserts some more information what a failure of a given test case means?

**Comment by sebastianmarzjan** [comment]
Plus, I'm thinking about a way to actually solve the instability of the test - at this stage, it's just changing the accounting

**Comment by sebastianmarzjan** [comment]
Let's :ship: and monitor if the issue was actually caused by test coupling

**Comment by idradm** [comment]
Merging to dev then

**Pull request merged**